src/constants/index.js

```diff
@@ -1,11 +1,18 @@
 /* eslint-disable import/prefer-default-export */
-export const TOTAL_ARCS = 20;
-
 export const COLOR_1 = '#ef0018';
 
+/**
+ * Maximum Length of Globe Curves that are drawn between Globe Points
```

Comment:
I personally like verbose config doc blocks like this so that in the future it's clear what a given piece of config does.

However, let me know if this is "too" verbose or if you'd prefer no notes here. I can also add comments to all the other constants if you'd like 🐒